test/src/code/code_test.dart

```diff
@@ -163,8 +165,8 @@ void main() {
             readonly
             readonly //readonly
             readonly // a readonly b'''
-            '\n\n'
-            '''
+              '\n\n'
+              '''
```

Comment:
?

Reply:
My formatter works differently. I'll look into it.